app/controllers/spree/admin/users_controller.rb

```diff
@@ -127,9 +129,13 @@ def new_email_unconfirmed?
         params[:user][:email] != @user.email
       end
 
+      def build_resource
+        model_class.new(locale: spree_current_user.locale)
```

Comment:
I think that it would be better to use the instance default here.

Reply:
Applied in 07353de7870a5321ab26d30df57e1861b67ffe86